lib/widgets/subscription_list.dart

```diff
@@ -175,10 +175,30 @@ class _SubscriptionList extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+
+    final sortedSubscriptions = List<Subscription>.from(subscriptions)
+    ..sort((a, b) {
+      if (a.pinToTop != b.pinToTop) {
+        return a.pinToTop ? -1 : 1;
+      }
+
+      if (a.isMuted != b.isMuted) {
+        return a.isMuted ? 1 : -1;
+      }
+
+      final isEmojiA = _startsWithEmoji(a.name);
+      final isEmojiB = _startsWithEmoji(b.name);
+      if (isEmojiA != isEmojiB) {
+        return isEmojiA ? -1 : 1;
+      }
+
+      return a.name.toLowerCase().compareTo(b.name.toLowerCase());
```

Comment:
This doesn't look like the right place to add sorting logic. We already sort by `pinToTop` and alphabetically, so it shouldn't be necessary to add code for that. Can you find the existing sorting logic and add to that?